docs/pages/guides/upload.mdx

```diff
@@ -0,0 +1,85 @@
+import { Callout } from 'nextra/components'
+
+# Uploading Books
+
+Stump has an opt-in feature that allows your users to upload books to your server.
+
+<Callout emoji="⚠️">
+	This functionality allows files to be uploaded to your server, so it should be used with caution.
+	This feature must be manually enabled by the server owner by setting the `enable_upload`
+	configuration variable to `true`.
+</Callout>
+
+## Enabling upload
+
+Uploading is **disabled by default**, and is gated behind the `enable_upload` configuration variable (`true` enables uploading). This variable can be set by modifying your `Stump.toml` file to set `enable_upload: true` or by setting the environment variable `STUMP_ENABLE_UPLOAD = true`.
```

Comment:
The suggested addition to the `Stump.toml` isn't valid toml:

```suggestion
Uploading is **disabled by default**, and is gated behind the `enable_upload` configuration variable (`true` enables uploading). This variable can be set by modifying your `Stump.toml` file to set `enable_upload = true` or by setting the environment variable `STUMP_ENABLE_UPLOAD=true`.
```

Reply:
Shit. I was thinking in yaml.